addons/project/models/project.py

```diff
@@ -719,48 +719,52 @@ def _get_stat_buttons(self):
             }),
             'show': True,
             'sequence': 2,
-        },
-        {
-            'icon': 'smile-o',
-            'text': _('Customer Satisfaction'),
-            'number': '%s %%' % (self.rating_percentage_satisfaction),
-            'action_type': 'object',
-            'action': 'action_view_all_rating',
-            'show': self.user_has_groups('project.group_project_rating') and self.rating_active and self.rating_percentage_satisfaction > -1,
-            'sequence': 5,
-        },
-        {
-            'icon': 'area-chart',
-            'text': _('Burndown Chart'),
-            'action_type': 'action',
-            'action': 'project.action_project_task_burndown_chart_report',
-            'additional_context': json.dumps({
-                'active_id': self.id,
-            }),
-            'show': self.user_has_groups('project.group_project_manager'),
-            'sequence': 7,
-        },
-        {
-            'icon': 'usd',
-            'text': _('Gross Margin'),
-            'number': format_amount(self.env, self.analytic_account_balance, self.company_id.currency_id),
-            'action_type': 'object',
-            'action': 'action_view_analytic_account_entries',
-            'show': self.user_has_groups('analytic.group_analytic_accounting'),
-            'sequence': 18,
-        },
-        {
-            'icon': 'users',
-            'text': _('Collaborators'),
-            'number': self.collaborator_count,
-            'action_type': 'action',
-            'action': 'project.project_collaborator_action',
-            'additional_context': json.dumps({
-                'active_id': self.id,
-            }),
-            'show': self.user_has_groups('project.group_project_manager'),
-            'sequence': 19,
         }]
+        if self.user_has_groups('project.group_project_rating'):
+            buttons.append({
+                'icon': 'smile-o',
+                'text': _('Customer Satisfaction'),
+                'number': '%s %%' % (self.rating_percentage_satisfaction),
+                'action_type': 'object',
+                'action': 'action_view_all_rating',
+                'show': self.user_has_groups('project.group_project_rating') and self.rating_active and self.rating_percentage_satisfaction > -1,
```

Comment:
Since we are sure the user has the groups, no need to again check it.
```suggestion
                'show': self.rating_active and self.rating_percentage_satisfaction > -1,
```